.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": true,
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 2017,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "node": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-console": 1
+  },
+  "plugins": [
+    "react"
+  ]
+}
```

Comment:
So have you implemented an eslint ?

Reply:
Yes, but it contains bare minimum rules although we can keep on adding more rules as needed. I haven't forced linting yet but we can discuss it. The commands for running lint can be found in server `README`